src/lavinmq/http/controller/vhosts.cr

```diff
@@ -33,11 +33,14 @@ module LavinMQ
           u = user(context)
           refuse_unless_administrator(context, u)
           name = URI.decode_www_form(params["name"])
+          body = parse_body(context)
+          tags = body["tags"]?.to_s.split(",") || [] of String
```

Comment:
I think we should remove white spaces
```suggestion
          tags = body["tags"]?.to_s.split(",").map(&:strip) || [] of String
```

Maybe also remove empty strings?

I noticed that in rabbitmq tag string `a,b,, c,d,` results in tag array `["a","b","","c","d",""]`, which I don't think need to mimic.

Reply:
```suggestion
          tags = body["tags"]?.to_s.split(',').map(&.strip).reject(&.empty?)
```